### PR TITLE
Remove expired references to IETF draft documents

### LINKED ISSUE #297: Expired drafts in references
https://github.com/tsvwg/ietf-multipath-dccp/blob/b8cc70de1761ddf446f1782fd8b0c728ac032a5d/draft-ietf-tsvwg-multipath-dccp.mkd?plain=1#L200-L206

This paragraph references expired drafts and external materials. While these might provide some interesting background information, as per my information there are no plans to progress those expired documents, hence, I would suggest to remove them and provide summery of the contents in this specification (as per of the introduction section or in the appendix). 

I think the ATSSS reference is good one to keep but then provide more text than just reference to the 3GPP TS. It is also perhaps OK to keep the references to the IETF slides and paper published on DCCP for further details.

### PR DESCRIPTION
**Pull request by markusa**
Closes #297 and #298

### CODE REVIEW

File: draft-ietf-tsvwg-multipath-dccp.mkd [hunk 1]
```diff
@@ -198,14 +195,11 @@ which is important for low-latency applications that do not require
 guaranteed delivery services, such as Audio/Video streaming.
 
 In addition to the integration into DCCP services, implementers or future specification could choose MP-DCCP for other use cases like
-3GPP 5G multi-access solutions (e.g., Access Traffic Steering, Switching, and Splitting (ATSSS) specified in {{TS23.501}}) or hybrid access networks
-{{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, in
-which MP-DCCP can be used for load balancing, seamless session handover and bandwidth aggregation
-using a framework such as {{I-D.amend-tsvwg-multipath-framework-mpdccp}}.
+3GPP 5G multi-access solutions (e.g., Access Traffic Steering, Switching, and Splitting (ATSSS) specified in {{TS23.501}}) or hybrid access networks which either combine a 3GPP and a non-3GPP access or a fixed and cellular access between user user-equipment/residential gateway and operator network. MP-DCCP can be used in these scenarios for load balancing, seamless session handover and bandwidth aggregation when non-DCCP traffic like IP, UDP or TCP is encapsulated into MP-DCCP.
```
Comment (gorryfair) [inline draft-ietf-tsvwg-multipath-dccp.mkd:198]: /which/that/?

**Review by gorryfair — CHANGES_REQUESTED** [review]
/which/that/?

File: draft-ietf-tsvwg-multipath-dccp.mkd (hunk 1 above)
Comment (markusa) [reply draft-ietf-tsvwg-multipath-dccp.mkd]: https://github.com/tsvwg/ietf-multipath-dccp/pull/307/commits/1533bd679543010e314bcb772d6e35711bc4b922

**Review by gorryfair — APPROVED** [review]
OK

**Review by zaheduzzaman — APPROVED** [review]

**Pull request merged**